pkg/sql/plan/base_binder.go

```diff
@@ -923,6 +923,24 @@ func bindFuncExprImplByPlanExpr(name string, args []*Expr) (*plan.Expr, error) {
 		} else {
 			return nil, moerr.NewInvalidArg(name+" function have invalid input args length", len(args))
 		}
+	case "unix_timestamp":
+		if len(args) == 1 {
+			if types.IsString(types.T(args[0].Typ.Id)) {
+				if exprC, ok := args[0].Expr.(*plan.Expr_C); ok {
+					sval := exprC.C.Value.(*plan.Const_Sval)
+					tp := judgeUnixTimestampReturnType(sval.Sval)
+					if tp == types.T_int64 {
+						args = append(args, makePlan2Int64ConstExprWithType(0))
+					} else {
+						args = append(args, makePlan2Float64ConstExprWithType(0))
```

Comment:
Actually we should use decimal here, to retain precision. However the current decimal code is buggy on converting to/from int/str.

When decimal reimplementation is complete, this part should be modified again.

Reply:
Yes, you are right. I remember this problem and will revise it to decimal later